docs/snippets/tracer/putMetadata.ts

```diff
@@ -3,6 +3,6 @@ import { Tracer } from '@aws-lambda-powertools/tracer';
 const tracer = new Tracer({ serviceName: 'serverlessAirline' });
 
 export const handler = async (_event: any, _context: any): Promise<void> => {
-    const res; /* ... */
-    tracer.putMetadata('paymentResponse', res);
+  const res; /* ... */
+  tracer.putMetadata('paymentResponse', res);
```

Comment:
Instead of passing a reference to the variable, pass directly an object with something like 
```
{
  'foo': 'bar'
}
```